src/org/zaproxy/zap/extension/pscanrulesBeta/ViewstateScanner.java

```diff
@@ -458,15 +465,15 @@ private void setVersion() {
         
         /* TODO once we have good Viewstate 1 & 2 parsers */
         public Object[] getObjectTree() throws Exception {
-        	throw new Exception("Not implemented (yet)");
+        	throw new Exception(Constant.messages.getString(MESSAGE_PREFIX + "exception"));

```

Comment:
The exception message should not be internationalised, it's not shown to the user.
Same comment for following exceptions.